com/course/zenkouv/src/main/java/lecture5/util/operation/Operation.java

```diff
@@ -0,0 +1,25 @@
+package lecture5.util.operation;
+
+public class Operation {
+
+    public double addition(double x, double y) {
+        return x + y;
+    }
+
+    public double subtraction(double x, double y) {
+        return x - y;
+    }
+
+    public double division(double x, double y) {
+        if (y == 0) {
+            System.out.println("Trying to divide by 0!");
+            return x;
```

Comment:
it is not correct to return x in this case
better  check `if (y == 0)` in Calculator and print error message there

Reply:
fix